consumer/src/test/java/no/nav/sbl/dialogarena/modiabrukerdialog/consumer/service/oppgavebehandling/LeggTilbakeOppgaveIGsakDelegateTest.java

```diff
@@ -73,7 +76,9 @@ private AnsattService mockAnsattService() {
     void skalLeggeTilbakeOppgaveIGsakUtenEndretTemagruppe() throws HentOppgaveOppgaveIkkeFunnet,
             LagreOppgaveOptimistiskLasing, LagreOppgaveOppgaveIkkeFunnet {
         WSHentOppgaveResponse hentOppgaveResponse = mockHentOppgaveResponseMedTilordning();
+        WSFinnOppgaveListeResponse finnOppgaveListeResponse = mockFinnOppgaveListe();
         when(oppgaveServiceMock.hentOppgave(any(WSHentOppgaveRequest.class))).thenReturn(hentOppgaveResponse);
+        when(oppgaveServiceMock.finnOppgaveListe(any(WSFinnOppgaveListeRequest.class))).thenReturn(finnOppgaveListeResponse);
```

Comment:
For `verify` linjen nedenfor; ` verify(oppgavebehandlingMock).lagreOppgave(lagreOppgaveRequestCaptor.capture());` så burde vi legge inn hvor mange ganger vi forventer at den blir kalt.

F.eks
```
 verify(oppgavebehandlingMock, times(3)).lagreOppgave(lagreOppgaveRequestCaptor.capture());
```